lms/djangoapps/philu_api/helpers.py

```diff
@@ -44,6 +45,16 @@ def get_social_sharing_urls(course_url, meta_tags):
     }
 
 
+def get_twitter_sharing_url(course_url, meta_tags, tweet_text):
```

Comment:
@muak50 after receiving above mentioned param in `get_social_sharing_urls` we can remove this function.